generate.js

```diff
@@ -1,6 +1,6 @@
 const fs = require('fs')
 const Mustache = require('mustache')
-const orgs = require('./out/data.json')
+const {orgs, timestamp} = require('./out/data.json')
```

Comment:
Can you load the data files modified date?

Also the modified date of the original json blob on the GCI server should be considered, if it gives appropriate headers.

Reply:
Yeah, it should get the modified date of the json files.

Use [`fs.stat`](https://nodejs.org/api/fs.html#fs_fs_statsync_path)